moped-editor/src/utils/activityLogFormatters/mopedFundingActivity.js

```diff
@@ -0,0 +1,42 @@
+import MonetizationOnOutlinedIcon from "@material-ui/icons/MonetizationOnOutlined";
+import DeleteOutlineIcon from "@material-ui/icons/DeleteOutline";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatFundingActivity = (change) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_proj_funding"];
+
+  let changeIcon = <MonetizationOnOutlinedIcon />;
+  let changeDescription = "Project funding updated";
+  let changeValue = "";
+
+  // add a new funding source
+  if (change.description.length === 0) {
+    changeDescription = "Added a new funding source ";
+    return { changeIcon, changeDescription, changeValue };
+  }
+
+  // delete existing record
+  if (change.description[0].field === "is_deleted") {
+    changeIcon = <DeleteOutlineIcon />;
+    changeDescription = "Funding source was deleted";
```

Comment:
Would you change this to "Deleted a funding source" ?

Reply:
passive voice was removed 🙂